packages/core/codegen.yml

```diff
@@ -0,0 +1,12 @@
+schema: "../../apps/subgraph/schema.graphql"
```

Comment:
We'll need to update this. Perhaps we should use a ts config and load from env?

Reply:
replaced the schema url. yml files don't support env variables and since it's a library that most probably won't need another subgraph scheme let's have it hardcoded.

if any contributors would like to change the schema we should let them know that they will have to replace the schema url although it seems that it should be implicitly trivial.